massgov/openmass
# Dp 24422 add the   did you find   feedback form to all collection and data view pages

**Description:**
### "Did you find what you were looking for on this webpage?" form on collection pages and listing pages.

- `Collection` page forms are built based on the first organization from the field_organizations field.
- `Data listing` pages take the form of the "Executive Office of Technology Services and Security" organization.


**Jira:** (Skip unless you are MA staff)
DP-24422


**To Test:**
- Collection pages
  - Edit any collection term and assign an organization (that shows a feedback form).
  - Visit the collection term page (collections/[url-name]) 
  - Notice the feedback form on the bottom of the page. 
  - Check the hidden inputs field68798989 and field57432673 have the id of the parent of the collection term.
  - Check the hidden input field47056299 contains the current URL. 
- Data listing pages
  - Visit any Data Listing page (/data-listing/all)
  - Notice the feedback form on the bottom of the page. 
  - Check the hidden inputs field68798989 and field57432673 have the following value: 560446
  - Check the hidden input field47056299 contains the current URL. 

---
### Example of feedback form /data-listing/all
![image](https://user-images.githubusercontent.com/3916979/169160660-24534c6e-331a-4290-8955-681a8b21737a.png)
---
### Example of feedback form /collections/eotss-product-and-service-catalog
![image](https://user-images.githubusercontent.com/3916979/169160830-78c6037d-7ba6-4098-94df-0b935f63009c.png)




---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
